StripeCore/StripeCore/Source/API Bindings/StripeAPI.swift

```diff
@@ -116,6 +117,8 @@ import PassKit
     /// American Express, Visa, Mastercard, Discover, Maestro.
     /// Japanese users can enable JCB by setting `JCBPaymentNetworkSupported` to YES,
     /// after they have been approved by JCB.
+    /// Users that have the Payment Method Cartes Bancaires set to Active, can enable it
+    /// by setting `CartesBancairesNetworkSupported` to YES.
```

Comment:
Is this true?  I'm not seeing a variable named `CartesBancairesNetworkSupported` that I can set to YES/True